### PR TITLE
linux6.10: Enable netkit support

### PR DESCRIPTION
**Pull request by tsndqst**
The BPF programmable network device was added in kernel 6.7.  Netkit device is used by [Cilium](https://www.cilium.io/) instead of `veth` interfaces to improve performance.  It would be nice to have this enabled in the default kernel as it would be a pain to recompile the kernel on my aarch64 Kubernetes cluster.

#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

**Comment by tsndqst** [comment]
This change can probably wait until the next kernel version bump.  I just wanted to request it now so it could be available some time in the future.

File: srcpkgs/linux6.10/files/i386-dotconfig [hunk 1]
```diff
@@ -2916,7 +2916,7 @@ CONFIG_TUN_VNET_CROSS_LE=y
 CONFIG_VETH=m
 CONFIG_VIRTIO_NET=m
 CONFIG_NLMON=m
-# CONFIG_NETKIT is not set
+CONFIG_NETKIT=y
```
Comment (vincele) [inline srcpkgs/linux6.10/files/i386-dotconfig:2919]: I'm not sure I'd bother with this one. Added it for consistency's sake?

Comment (tsndqst) [reply srcpkgs/linux6.10/files/i386-dotconfig:2919]: Pretty much.

**Comment by leahneukirchen** [comment]
Drop the revision bump then we can pick it up on next minor kernel release.

**Comment by tsndqst** [comment]
Updated.
Thanks @leahneukirchen

**Pull request merged**